cpp/include/cugraph/experimental/graph.hpp

```diff
@@ -22,18 +22,21 @@
 #include <rmm/device_uvector.hpp>
 
 #include <cstddef>
+#include <optional>
 #include <string>
 #include <type_traits>
 #include <vector>
 
 namespace cugraph {
 namespace experimental {
 
+// FIXME: consider using std::optional for optional parameters (i.e. weights & segment_offsets)
+
```

Comment:
Is this comment still relevant? I think you changed them to optional already right?

Reply:
Good catch!!! No, I will delete.